src/main/kotlin/no/nav/syfo/infrastructure/clients/dokarkiv/dto/Dokumentvariant.kt

```diff
@@ -0,0 +1,63 @@
+package no.nav.syfo.infrastructure.clients.dokarkiv.dto
+
+enum class FiltypeType(
+    val value: String,
+) {
+    PDFA("PDFA"),
+}
+
+enum class VariantformatType(
```

Comment:
Samme her? 😊 Kunne den hetet `DokumentFormat` eller noe annet? Hvis ikke er `Variantformat` fint
```suggestion
enum class Variantformat(
```